folium/map.py

```diff
@@ -297,6 +296,27 @@ def render(self, **kwargs):
             '</style>'), name='map_style')
 
         super(LegacyMap, self).render(**kwargs)
+    
+    @property
+    def location(self):
+        return self._location
```

Comment:
That's a fair point. I was more worried about changing anything in an `__init()__` function, since it seems like the property decorators leave those alone, but it does result in unnecessary code. I could swap out for a.... class function?